src/main/java/org/cryptomator/ui/health/StartController.java

```diff
@@ -80,7 +77,7 @@ private void loadKey() {
 	}
 
 	private void verifyVaultConfig(KeyLoadingStrategy keyLoadingStrategy) throws VaultConfigLoadException {
-		var unverifiedCfg = unverifiedVaultConfig.get();
+		var unverifiedCfg = vault.getUnverifiedVaultConfig();
```

Comment:
Not worth the change effort, imho.

Since we are not in the `PerVault`-Scope, we need a getter for the ConfigCache in the Vault class and additional provider in the healthModule. In 76811f9 i created a getter, inject the vault, but only to set the config cache to a final variable